data/json/vitrification/vitrified_farm.json

```
@@ -0,0 +1,506 @@
+        "!**********************E",
+        "!*    *    **    *    *E",
+        "!* 7  7  7 ** 7  q  7 *E",
+        "!*     r   **         *E",
+        "!*         **         *E",
+        "!**7     7 ***7     7**E",
+        "!*         **         *E",
+        "!*         **         *E",
+        "!* 7  7  7 ** 7  7  7 *E",
+        "!*    *    **    *    *E",
+        "!**********************E",
+        "EEEEEEEEEEEEEEEEEEEEEEEE"
+      ],
+      "palettes": [ "orchard_vitrified_palette" ],
+      "terrain": { "E": "t_vitrified_edge", "q": "t_vitrified_soil_talker", "!": "t_vitrified_fence_barbed" },
+      "nested": { "r": { "chunks": [ "statue_vitrified_1x1" ] } }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "vitrified_orchard_sky",
+    "object": {
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "rows": [
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "                       E",
+        "EEEEEEEEEEEEEEEEEEEEEEEE"
+      ],
+      "terrain": { "E": "t_vitrified_edge" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "vitrified_sky",
+    "object": {
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "rows": [
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE",
+        "EEEEEEEEEEEEEEEEEEEEEEEE"
+      ],
+      "terrain": { "E": "t_vitrified_roof" }
+    }
+  },
```

```suggestion
  {
    "type": "mapgen",
    "method": "json",
    "update_mapgen_id": "vitrified_sky",
    "object": {
      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
      "set": [ { "square": "terrain", "id": "t_vitrified_roof", "x": 0, "y": 0, "x2": 23, "y2": 23 } ]
    }
  },
```
Very much an unnecessary change but I just thought I'd make you aware of this field for future usage and that being able to omit rows if you're only doing stuff by coords is cool ^^ (Not that I'm suggesting using it where it reduces readability like)